src/styles/FarmCard.css

```diff
@@ -0,0 +1,109 @@
+* {
+  box-sizing: border-box;
+}
+
+.farm-card {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  width: 360px;
+  height: 330px;
+}
+
+.farm-card__top {
+  position: relative;
+  background-color: #f0f5fb;
```

Comment:
update all your colors `var(--ks-color-name)` 

you may need to pull and rebase master to get these